app/Controllers/Http/PostsController.ts

```diff
@@ -0,0 +1,85 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import { schema, rules } from '@ioc:Adonis/Core/Validator'
+import Hash from '@ioc:Adonis/Core/Hash'
+
+import User from 'App/Models/User'
+import Profile from 'App/Models/Profile'
+
+export default class PostsController {
+  public async createUser({ request, response }: HttpContextContract) {
+    try {
+      const newUserSchema = schema.create({
+        email: schema.string({}, [rules.email()]),
+        password: schema.string({}, [rules.minLength(8), rules.maxLength(16), rules.alphaNum()])
+      })
```

Comment:
Instead of creating validation schema in controller, You can use validator classes instead.

Ref: https://docs.adonisjs.com/guides/validator/introduction#validator-classes

Reply:
Done